lua/Navigator/navigate.lua

```diff
@@ -13,18 +13,27 @@ local N = {
 ---Detect and load correct mux
 ---@return Vi
 local function load_mux()
+    local ok_zellij, zellij = pcall(function()
+        return require('Navigator.mux.zellij'):new()
+    end)
+    if ok_zellij then
+        return zellij
+    end
```

Comment:
It would be better to move this after tmux, as I believe most people still prefers tmux.